src/samples/flow/iio/sol-flow-intel-minnow-max-linux_gt_3_17.json

```diff
@@ -64,6 +64,14 @@
            "buffer_size": "-1"
      },
      "type": "iio/thermometer"
+  },
+  {
+     "name": "MyProximity",
+     "options": {
+           "iio_device": "apds9960 create,i2c,platform/80860F41:05,0x39,apds9960 0",
+           "buffer_size": "-1"
```

Comment:
@barbieri I found not all sensors support the buffering/trigger reading. In this case, the apds9960 does not support it